src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <!-- The version is frozen to allow MSBuild.exe loading it into the default load context -->
+    <AssemblyVersion>8.0.100.0</AssemblyVersion>
```

Comment:
Shouldn't this be 9.0.100 since main is .NET 9?

Reply:
Just like https://github.com/dotnet/installer/pull/17732, I would like this to appear in 8.0.2xx / VS 17.9. So the version from 8.0.2xx onwards would ideally be 8.0.200.0 or 8.0.100.0. But it's fine to make it 9.0.100 if you prefer that; any version will do as long as it's fixed.

Reply:
I'm not sure if `$(VersionPrefix)` would help. I'm probably misunderstanding but having different branches produce different versions is specifically what I don't want. VS main currently has 8.0.100 of this assembly and what I basically want to avoid is having to bump the reference in MSBuild every time a new version of SDK is inserted. So my goal is for the version of the file in VS to never change, similar to how MSBuild has frozen its assembly versions to 15.1.0.0.

> Also, are there any risk with having the stable version? Does anything rely on loading the assembly dynamically?

The file is loaded only by MSBuild, to my best knowledge.